src/Compilers/Core/Portable/Syntax/InternalSyntax/SyntaxList.WithManyChildren.cs

```diff
@@ -122,11 +107,6 @@ private bool HasNodeTokenPattern()
 
         internal sealed class WithManyChildren : WithManyChildrenBase
         {
-            static WithManyChildren()
```

Comment:
Same question below. Can we remove `internal WithManyChildren(ObjectReader reader)`?